src/llama-vocab.cpp

```diff
@@ -340,6 +340,7 @@ struct llm_tokenizer_bpe : llm_tokenizer {
                     "'s|'t|'re|'ve|'m|'ll|'d| ?\\p{L}+| ?\\p{N}+| ?[^\\s\\p{L}\\p{N}]+|\\s+(?!\\S)",
                 };
                 break;
+            case LLAMA_VOCAB_PRE_TYPE_EXAONE4:
             case LLAMA_VOCAB_PRE_TYPE_GPT2:
```

Comment:
```suggestion
            case LLAMA_VOCAB_PRE_TYPE_GPT2:
```
Let's remove this, it will make merge simpler.